.storybook/preview.js

```diff
@@ -75,7 +75,12 @@ export const decorators = [
 		}
 
 		return (
-			<WithTestRegistry features={ features } route={ route }>
+			<WithTestRegistry
+				features={ features }
+				route={ route }
+				// Expose registry as global for tinkering.
+				callback={ ( registry ) => ( global.registry = registry ) }
```

Comment:
This is minor, but I'm wary of using implicit return when we aren't actually intending to use the return value. It _implies_ the return value matters is all. Instead, let's do:

```suggestion
				callback={ ( registry ) => {
					global.registry = registry;
				} }
```